Src/Newtonsoft.Json/Newtonsoft.Json.csproj

```diff
@@ -33,6 +33,13 @@
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
     <CodeAnalysisRuleset>Newtonsoft.Json.ruleset</CodeAnalysisRuleset>
     <EmbedUntrackedSources>true</EmbedUntrackedSources>
+    <!--Package Validation-->
+    <EnablePackageValidation>true</EnablePackageValidation>
+    <PackageValidationBaselineVersion>13.0.1</PackageValidationBaselineVersion>
+    <!--CP0003 will be logged when the assembly identity between baseline package and current don't match.
+    Disable that warning since local builds do not strong-name sign the assembly, causing a mistmatch when comparing
+    the previously shipped assemblies and current.-->
+    <NoWarn>$(NoWarn);CP0003</NoWarn>
```

Comment:
Can we apply a condition here so that it's not permitted during an official build?  Maybe `Condition='$(SignAssembly)' != 'true'"`

Reply:
Oh good suggestion, sure thing, will do.